formal-spec/Leios/Foreign/Defaults.agda

```diff
@@ -15,22 +18,28 @@ open Equivalence
 
 -- The module contains very simple implementations for the functionalities
 -- that allow to build examples for traces for the different Leios variants
-module Leios.Trace where
+module Leios.Foreign.Defaults where
+
+numberOfParties : ℕ
+numberOfParties = 1
```

Comment:
I feel like this might be trouble later, since you cannot have a block that hasn't been made by the SUT.

Reply:
Yes, agree. The `numberOfParties` is used in the stake distribution. 
For testing it should be at least 2, the system under test and an other party